obal/data/roles/build_package/tasks/brew_wait_and_download.yml

```diff
@@ -1,13 +1,10 @@
 ---
```

Comment:
I think we could rename this file `wait_and_download` ? Or split the waiting and the downloading up

Reply:
Separate wait and download sounds good.  I wasn't sure how far to take it in this one since my next pr is to make it brew/koji agnostic